beacon-chain/rpc/service.go

```diff
@@ -0,0 +1,81 @@
+package rpc
+
+import (
+	"context"
+	"fmt"
+	"net"
+
+	pb "github.com/prysmaticlabs/prysm/proto/beacon/rpc/v1"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+)
+
+var log = logrus.WithField("prefix", "rpc")
+
+// Service hi.
```

Comment:
Is there a reason we put `hi` there? and for all the comments too

Reply:
Whoops, forgot to change lol